unstoppable-ios-app/domains-manager-ios/Modules/Home/Activity/WalletTransactionDisplayInfo.swift

```diff
@@ -116,3 +118,25 @@ extension WalletTransactionDisplayInfo {
         }
     }
 }
+
+import UIKit
+
+extension WalletTransactionDisplayInfo {
+    var chainFullName: String {
+        if let blockchainType = BlockchainType(rawValue: chain) {
```

Comment:
The issue is much deeper. 
We used to have one enum that reflected blockchains we're working with: ETH, MATIC, _ZIL_. It is due to the fact that our domains only existing on this chains.
Now, we have different set of supported chains for number of cases. 
- All wallets now also include balances and txs for Base chain
- ULW also support BTC and SOL
- Send crypto feature by imported wallets has own set of supported chains
It is not possible to mix them, and therefore there's places where multiple use cases collides.
In this particular case, tx details, they can be from chains that our BlockchainType does not cover. And this functions abstracting it so no other places in the app aware of this details.
I'm open to suggestions how it can be improved in this particular case without mutating existing core structures like BlockchainType. It seems to me the issue should be resolved on the level deeper.

Reply:
@Oleg-Pecheneg agreed to make a deeper refactoring. Please create me a ticket, we'll discuss details tomorrow. Thanks!